### PR TITLE
Bump WP tested up to tag to 5.9

### PR DESCRIPTION
**Pull request by rodrigoprimo**
I tested just a few of the main MailPoet pages and confirmed that they are working. I'm trusting the acceptance tests would have caught issues if there are any. I also read the [WP 5.9 field guide](https://make.wordpress.org/core/2022/01/10/wordpress-5-9-field-guide/) and couldn't find anything highlighted there with the potential to impact us.

[MAILPOET-4085]

[MAILPOET-4085]: https://mailpoet.atlassian.net/browse/MAILPOET-4085?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brezocordero — DISMISSED** [review]

**Review by veljkho — APPROVED** [review]

**Comment by rodrigoprimo** [comment]
@veljkho, I see that you approved this PR but did not merge it. Should I go ahead and merge it? I will need a new approval as there was a conflict that I just fixed that made all the reviews stale.

**Review by veljkho — DISMISSED** [review]

**Comment by veljkho** [comment]
Hey @rodrigoprimo , I was waiting for you to fix the conflict... I see it again.

**Review by veljkho — APPROVED** [review]

**Comment by veljkho** [comment]
@rodrigoprimo yes for sure I will ping you as always, this time I was busy with the release so forgot to ask you, no problem.

**Pull request merged**
